src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,9 +1,40 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "products")
 public class Product {
-    private Long id;
+
+
+    @Id
+    @NotNull
+    @GeneratedValue
+    @Column(name = "PRODUCT_ID", unique = true)
+    private Long Id;
+
+    @Column(name = "NAME")
     private String name;
+
+    @Column(name = "DESCRIPTION")
     private String description;
-    private Double price;
-    private Long groupId;
+
+    @Column(name = "PRICE")
+    private BigDecimal price;
+
+    //@ManyToOne
+    //@Column(name = "GROUP_ID")
+    //private GroupId groupId;
+
+    //@Column(name = "VERSION")
```

Comment:
Nie powinno być komentarza

Reply:
Ale to normalnie zostawić i stworzyć tylko Klase GroupId, żeby nie wyświetlało błędu?

@Column(name = "GROUP_ID")
    private GroupId groupId;

    @Column(name = "VERSION")